application/event_processor/dos.py

```diff
@@ -67,80 +68,119 @@ def __repr__(self) -> str:
         else:
             name = "NO-VALID-NAME"
 
-        return f"<uid={self.uid} ods={self.odscode} type={self.typeid} status={self.statusid} name='{name}'>"
+        return (f"<DoSService: name='{name[0:16]}' id={self.id} uid={self.uid} "
```

Comment:
Why is the name limited to 16/17 characters?

Reply:
The repr function is used to easily identify a python object with a string. For logs or local testing. This reduces verbose strings from appearing if the pharmacy happens to have a very very long name